JitsiMeetJS.js

```diff
@@ -268,7 +268,11 @@ export default {
         window.connectionTimes['obtainPermissions.start']
             = window.performance.now();
 
-        return RTC.obtainAudioAndVideoPermissions(options || {})
+        const obtainMediaPromise = RTCBrowserType.usesNewGumFlow()
+            ? RTC._newObtainAudioAndVideoPermissions(options)
```

Comment:
i think i added this leading `_` and we should probably get rid of it since it's called from outside that file.